server/graphql/v2/object/ExpenseValuesByRole.ts

```diff
@@ -0,0 +1,46 @@
+import { GraphQLNonNull, GraphQLObjectType } from 'graphql';
+import { GraphQLNonEmptyString } from 'graphql-scalars';
+
+import { Expense } from '../../../models';
+import { ExpenseDataValuesRoleDetails } from '../../../models/Expense';
+import { getIdEncodeResolver, IDENTIFIER_TYPES } from '../identifiers';
+
+import GraphQLAccountingCategory from './AccountingCategory';
+
+const GraphQLExpenseValuesRoleDetails = new GraphQLObjectType({
+  name: 'ExpenseValuesRoleDetails',
+  fields: () => ({
+    accountingCategory: {
+      type: GraphQLAccountingCategory,
+      resolve: async (data: ExpenseDataValuesRoleDetails, _, req) => {
+        // Try to load the accounting category from the DB, fallback to the value stored in data (in case it was deleted)
+        if (data?.accountingCategory?.id) {
+          const category = await req.loaders.AccountingCategory.byId.load(data.accountingCategory.id);
+          return category || data.accountingCategory;
+        }
+      },
+    },
+  }),
+});
+
+export const GraphQLExpenseValuesByRole = new GraphQLObjectType({
+  name: 'ExpenseValuesByRole',
+  fields: () => ({
+    id: { type: new GraphQLNonNull(GraphQLNonEmptyString), resolve: getIdEncodeResolver(IDENTIFIER_TYPES.EXPENSE) },
+    submitter: {
```

Comment:
Ideally, these fields are also defined by constants.

Reply:
They're actually not named the same way: it's `collectiveAdmin` in the DB and `accountAdmin` in graphql